cookbook/specs/movement.json

```diff
@@ -0,0 +1,87 @@
+{
+    "proposal": {
+        "title": "Add Specs: Movement",
+        "description": "Adding new specification support for relaying Movement data on Lava",
+        "specs": [
+            {
+                "index": "MOVEMENT",
+                "name": "movement mainnet",
+                "enabled": true,
+                "imports": [
+                    "APT1"
+                ],
+                "reliability_threshold": 268435455,
+                "data_reliability_enabled": true,
+                "block_distance_for_finalized_data": 0,
+                "blocks_in_finalization_proof": 5,
+                "average_block_time": 200,
+                "allowed_block_lag_for_qos_sync": 50,
+                "shares": 1,
+                "min_stake_provider": {
+                    "denom": "ulava",
+                    "amount": "47500000000"
+                },
+                "api_collections": [
+                    {
+                        "enabled": true,
+                        "collection_data": {
+                            "api_interface": "rest",
+                            "internal_path": "",
+                            "type": "GET",
+                            "add_on": ""
+                        },
+                        "apis": [],
+                        "headers": [],
+                        "inheritance_apis": [],
+                        "parse_directives": [],
+                        "verifications": [
+                            {
+                                "name": "chain-id",
+                                "values": [
+                                    {
+                                        "expected_value": "4"
+                                    }
+                                ]
+                            },
+                            {
+                                "name": "pruning",
+                                "values": [
+                                    {
+                                        "latest_distance": 21600
```

Comment:
for 200msec blocks pruning verification should be:
24*60*60/0.2 = 432000

Reply:
@Yaroms  because movement gets the most of Aptos. They had the same speed as blocks. However on the aptos spec it is then not correct either. I changed that too now.